### DISCUSSION TITLE
Updated plot book

### DISCUSSION THREAD

**Discussion by IsocratesGrey** [endless-sky/endless-sky#7810] [Lore]
I made some updates to the plot analysis I built based on #7140. The idea of it is that this can help people to create new plotlines that fit into the mold created by the free worlds campaign.

The excel file attached has four sections: 
Starting Campaign: an overview of the starting campaign (Free Worlds) broken down into events and missions and how many missions per section. I've also got the story beats listed there.
Quest Lines: an overview of most of the quest lines you can follow in the game, it's not done yet as I haven't added in the Remnant
Plot Beats: a quick overview of what happens in the starting campaign broken into the chapters and connected to major events, this is the main story beats section for those building new campaigns and it also lists about how many missions (including sub missions) are undertaken at each plot beat
Missions by chapter: For those wanting to just know approximately how long each chapter is, it's 30-40 missions per chapter and 10 missions in the prologue.

Please let me know if there are errors. I wasn't sure if I should count sub missions (1b,c,d etc) as a single or separate missions, so in general they're counted individually.

[Endless Sky Plot Book.xlsx](https://github.com/endless-sky/endless-sky/files/10189447/Endless.Sky.Plot.Book.xlsx)

Here's the plot beats in a table because that's probably the most important to writers for quick access. 

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/narney/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/narney/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

Major   Events | FW Quest Line | # Missions | Story Beats | Plotline Trigger | Plot
-- | -- | -- | -- | -- | --
Remeberance Day | Intro | 0 | Pre campaign: Introduce Alphas | Date | Game Intro
War Begins | Intro | 0 | Pre campaign: War starts, nukes   fall | Date | Game Intro
Initial Deployment | Free Worlds Prologue | 8 | Campaign Prologues | Date | Prologue Body
Capture of Kornephoros | Free Worlds Prologue | 2 | Inciting Incident: Capture of   Kornephoros | Chosen Sides | Prologue Climax
recapture of Kornephoros, Oathkeepers   Founded | Defend the Free Worlds | 4 | Open of Chapter 1 | Plot Chapter 1 | Chapter 1 Intro
Tarazed neutrality | Defend the Free Worlds | 4 | Player meeting major players in   campaign | Plot Chapter 1 | Chapter 1 Intro
fw conservatory founded | Defend the Free Worlds | 13 | Player impacts galaxy, access to   faction weapon 1 | Plot Chapter 1 | Chapter 1 Body
Wolfpack Reveal, Navy Occupies South | Defend the Free Worlds | 7 | Close of chapter 1, major reveal   of sub factions | Plot Chapter 1 | Chapter 1 Climax
navy using mark ii ships | Freeworlds Expansion, Alphas | 9 | Open of chapter 2 | Plot Chapter 2 | Chapter 2 Intro
fw southern expansion | Freeworlds Expansion, Alphas | 3 | Player climbs the ranks, makes   decisions impacting faction, access to faction weapon 2 | Plot Chapter 2 | Chapter 2 Intro
fw occupying the north | Freeworlds Expansion, Alphas | 2 | Campaign break | Plot Chapter 2 | Chapter 2 Intro
fw northern expansion | Freeworlds Expansion, Alphas | 9 | Player climbs the ranks | Plot Chapter 2 | Chapter 2 Body
alphas catpure Poisonwood/Liberation/NPC   death | Freeworlds Expansion, Alphas | 3 | Return of the Alphas, making   consequences real | Plot Chapter 2 | Chapter 2 Body
Bloodsea joins/taken by FW | Freeworlds Expansion, Alphas | 3 | Player impacts galaxy, acess to   faction ship | Plot Chapter 2 | Chapter 2 Body
dreadnought deployment | Freeworlds Expansion, Alphas | 1 | Faction gaining strength | Plot Chapter 2 | Chapter 2 Body
fw expanded and cut | Freeworlds Expansion, Alphas | 7 | Player climbs the ranks | Plot Chapter 2 | Chapter 2 Body
FW Defend New Tibet | The real threat | 1 | Climax of chapter: Potential   Nuclear Reveal | Plot Chapter 2 | Chapter 2 Climax
fw at war with Syndicate or republic | The real threat | 1 | Close of chapter 2: a new war | Plot Chapter 2 | Chapter 2 Climax
fw Syndicate/Republic war | Fight the Republic/Syndicate | 15 | Chapter 3A: Player fills major   role in the war FW/Rp/Sy | Plot Chapter 3 | Chapter 3A Body
syndicate occupies/FW attacks sol | Fight the Republic/Syndicate | 1 | Someone attacks Sol (end ch 3A) | Plot Chapter 3 | Chapter 3A Climax
pug invasion | Repel Pug Invasion | 20 | War with the Pug, access to jump drive and faction weapon/equipment 3| Plot Chapter 3 | Chapter 3B Body
battle of algenib | Syndicate Alpha War | 4 | Ch 3 Climax: Defeat the Alpha   Threat | Plot Chapter 3 | Chapter 3B Climax
mission "Geminus Rebuilt" | Free Worlds Epilogue | 2 | Main Plot Epilogue | Plot Chapter 3 | Epilogue
  | Free Worlds Epilogue | 3 | Main Plot Epilogue | Epilogue | Epilogue

</body>

</html>

**Comment by Zitchas** [comment]
Could be a useful reference.

**Comment by dahe-at-google** [comment]
Looks great - thanks for doing this.
